Mac/README.rst

```diff
@@ -120,6 +120,8 @@ support ppc (Xcode 4 on 10.6 and later systems).  The flavor can be specified
 using the configure option ``--with-universal-archs=VALUE``. The following
 values are available:
 
+  * ``universal2``: ``arm64``, ``x86_64``
```

Comment:
Should not `all` also contain `arm64`?

Reply:
I don't want to change the meaning of existing sets.   It is also not possible to build fat binaries with arm64 and ppc or i386 with a single compiler, the current compiler in Xcode only supports arm64 and x86_64.

Reply:
For 3.10 we can reconsider the sets of architectures to support, possibly defaulting to "universal2" and only keeping that and "intel" (for folks still needing 32-bit support).  But that's for another issue.